build.gradle

```diff
@@ -297,6 +302,23 @@ tasks.withType(GenerateModuleMetadata) {
     enabled = false
 }
 
+dependencyLocking {
+    lockAllConfigurations()
```

Comment:
is there a flag for including transitive dependencies?

Reply:
i think transitives are included, it's just not structured like how nebula does it :/

Reply:
ah i see cool. there shouldnt be any 3rd party libs published with dynamic or snapshot dependencies in their poms anyway